storage/csv.go

```diff
@@ -196,6 +194,24 @@ func (c *CSVBatch) PersistModel(ctx context.Context, m interface{}) error {
 				continue
 			}
 
+			var encodeAsJSON bool
+
+			// Strings marked as json type are assumed to already be encoded
+			if fk != reflect.String && (t.types[i] == "json" || t.types[i] == "jsonb") {
+				encodeAsJSON = true
+			} else if fk == reflect.Interface {
+				encodeAsJSON = true
+			}
```

Comment:
This means slices of strings/bytes/structs will be encoded as json?

Reply:
Slices of strings encode as json. For example `MultisigTransaction.Approved` is a `[]string` with no column type tag. go-pg assigns that a json type by default, which matches our schema. Same with `MinerInfo.ControlAddresses` and `MinerInfo.MultiAddresses`